README.markdown

```diff
@@ -285,6 +428,20 @@ var diameter: Double {
 }
 ```
 
+### Final
+
+Mark classes `final` when inheritance is not intended. Example:
+
+```swift
+// We don't allow subclassing of an equilateral triangle
+final class EquilateralTriangle : Triangle {
+    let sideLength: Int
+    init(sideLength: Int) {
+        self.sideLength = sideLength
+    }
+}
+```
```

Comment:
So I think that this info could get covered in the task outlined in issue #20. I've added these comments there so they get captured. Feel free to add more there if you'd like.